automation/catch-uncommitted-output

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+npm run build
+
+if ! git diff HEAD --exit-code ; then
+	echo '\n** Uncommitted changes found after build - FAIL **'
+	exit 1
+elif test -n "$(git ls-files --exclude-standard --others)" ; then
```

Comment:
It'd be nice if we could avoid duplication of `git ls-files --exclude-standard --others` here